bitnami/postgresql/values.yaml

```diff
@@ -1375,6 +1375,34 @@ backup:
     ## ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
     ##
     tolerations: []
+    ## @param backup.cronjob.podAffinityPreset PostgreSQL backup pod affinity preset. Ignored if `backup.cronjob.affinity` is set. Allowed values: `soft` or `hard`
+    ## ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#inter-pod-affinity-and-anti-affinity
+    ##
+    podAffinityPreset: ""
```

Comment:
I'm missing the `podAntiAffinityPreset` key so it is consistent with the others `affinity` and `antiAffinity` settings in this `values.yaml` files. Can you add it?